hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/tasks/TraceabilityExportTask.java

```diff
@@ -163,13 +163,18 @@ public SystemTaskResult process(final long literalNum, final Instant now) {
         throw new UnsupportedOperationException();
     }
 
-    private boolean needsBackPressure(final Instant now, final MerkleNetworkContext curNetworkCtx) {
-        return inHighGasRegime(now)
+    private boolean needsBackPressure(final MerkleNetworkContext curNetworkCtx) {
+        return inHighGasRegime()
                 || curNetworkCtx.getEntitiesTouchedThisSecond() >= dynamicProperties.traceabilityMaxExportsPerConsSec();
     }
 
-    private boolean inHighGasRegime(final Instant now) {
-        return handleThrottling.gasLimitThrottle().freeToUsedRatio(now)
+    private boolean inHighGasRegime() {
+        return handleThrottling
+                        .gasLimitThrottle()
+                        .freeToUsedRatio(handleThrottling
+                                .gasLimitThrottle()
+                                .usageSnapshot()
+                                .lastDecisionTime())
```

Comment:
Myself, I would have pulled out this nested fluid callchain in order to give this value a name, and simplify the cognitive load on the reader.